lib/hci-socket/bindings.js

```diff
@@ -214,8 +214,8 @@ NobleBindings.prototype.onLeConnComplete = function(status, handle, role, addres
     this._gatts[handle].exchangeMtu(256);
   } else {
     uuid = this._pendingConnectionUuid;
-
-    error = new Error(Hci.STATUS_MAPPER[status] || ('Unknown (' + status + ')'));
+    var errorCode = status.toString(16);
+    error = new Error('HCI Error: (0x' + errorCode +  ') ' + Hci.STATUS_MAPPER[status] || ('HCI Error: Unknown (' + status + ')'));
```

Comment:
Two questions:

* should we put the hex code after the string?
* should we also use hex code for unknown errors?